app/views/videos/show.html.haml

```diff
@@ -11,3 +11,31 @@
         .actions
           %a.btn.btn-primary(href="") Watch Now
           %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for([@video, Review.new], label_col: "col-sm-3") do |f|
+        %fieldset
+          .form-group
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map {|number| [pluralize(number, "Star")]}), { label: "Rate this video" }
```

Comment:
`[5,4,3,2,1].map {|number| [pluralize(number, "Star")]}` this is a great candidate for a helper method :) Later on you will see why.